quickwit-metastore/src/metastore/file_backed_metastore/index.rs

```diff
@@ -205,10 +206,21 @@ impl Index {
             });
         }
 
-        self.index.update_timestamp = now_timestamp;
         Ok(())
     }
 
+    /// Helper to publish a list of splits.
```

Comment:
I believe the helper function is actually the one above.

Reply:
good point